rust/tools/authorization-logic/src/souffle/datalog_ir.rs

```diff
@@ -18,14 +18,20 @@
 //! this authorization logic into datalog simpler.
 use crate::ast::*;
 
+#[derive(Clone)]
+pub enum DLIRRvalue {
+    PredicateRvalue { predicate: AstPredicate },
+    BinopRValue { lnum: String, binop: AstBinop, rnum: String }
```

Comment:
It looks like there is inconsistent casing of `Rvalue`/`RValue` here.

Reply:
fied